source/_components/aws.markdown

```diff
@@ -0,0 +1,186 @@
+---
+layout: page
+title: "Amazon Web Services"
+description: "Instructions on how to integrate Amazon Web Services with Home Assistant."
+date: 2019-03-21 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: aws.png
+ha_category:
+  - Notifications
+ha_release: "0.91"
+---
+
+The `aws` integration provides a single place to interact [Amazon Web Services](https://aws.amazon.com/). Current it provide a notification platform can send a message to [AWS SQS](https://aws.amazon.com/sqs/), [AWS SNS](https://aws.amazon.com/sns/), or invoke [AWS Lambda](https://aws.amazon.com/lambda/) functions.
+
+## {% linkable_title Setup %}
+
+You have to have a AWS account to use Amazon Web Services, create one [here](https://aws.amazon.com/free/) with 12 months free tier benefit. Please note, even in the first 12-months, you may still be billing if you use some resources out of limit, we advice you closely monitoring your costs in [AWS Billing Console](https://console.aws.amazon.com/billing/). You can read [Control your AWS costs](https://aws.amazon.com/getting-started/tutorials/control-your-costs-free-tier-budgets/) guide to get more information.
+
+The `lambda`, `sns` and `sqs` services used in `aws` components all provided **Always Free** tier for all users even after 12-months period. The general usages in Home Automation most likely would not reach the free tier limit. Please read [Lambda Pricing](https://aws.amazon.com/lambda/pricing/), [SNS Pricing](https://aws.amazon.com/sns/pricing/) and [SQS Pricing](https://aws.amazon.com/sqs/pricing/) for more details.
+
+`aws` component is using [botocore](https://botocore.amazonaws.com/v1/documentation/api/latest/index.html) underneath to communicate with Amazon Web Services, which is also used in [AWS Command Client Interface](https://aws.amazon.com/cli/) tool. Therefore, `aws` shared same credential and profiles with `awscli` tool. Please read [Configuring the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) to learn how to get access keys and security manage them in your local system.
+
+## {% linkable_title Configuration %}
+
+To use `aws` component and `notify` platform in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+aws:
+  credentials:
```

Comment:
missing: `region_name: us-east-1`?

Reply:
No,`region_name` is not required for `credential`, but required for notify service. e.g. credential is region independent.